packer/templates/windows_2008_r2.json

```diff
@@ -216,6 +258,8 @@
     "autounattend": "{{template_dir}}/../answer_files/2008_r2/Autounattend.xml",
     "scripts_dir": "{{template_dir}}/../../scripts",
     "resources_dir": "{{template_dir}}/../../resources",
-    "box_version": "0.1.0"
+    "switch_name": "langrill.local",
```

Comment:
This looks suspiciously like it is specific to your installation, is there a guaranteed valid value to default this too? Can this be removed and a valid default will be automatically selected as virtualbox and vmware builders do?